src/Form/EntryForm.php

```diff
@@ -18,14 +20,22 @@ class EntryForm extends FormBase {
   /**
    * @var \Drupal\tfa\TfaManager
    */
-  protected $tfaManager;
-
-  public function __construct(TfaManager $tfa_manager) {
-    $this->tfaManager = $tfa_manager;
+  protected $tfaValidationManager;
+  protected $tfaLoginManager;
+  protected $tfaValidationPlugin;
+  protected $tfaLoginPlugin;
+  protected $tfaFallbackPlugin;
+
+  public function __construct(TfaValidationPluginManager $tfa_validation_manager, TfaLoginPluginManager $tfa_login_manager) {
+    $this->tfaValidationManager = $tfa_validation_manager;
+    $this->tfaLoginManager = $tfa_login_manager;
   }
 
   public static function create(ContainerInterface $container) {
-    return new static($container->get('tfa.manager'));
+    return new static(
+      $container->get('plugin.manager.tfa.validation'),
```

Comment:
They will be added, but not until the integrations are ready. This part is a work in progress.